### ISSUE TITLE
documentation demonstrates incorrect key generation

### ISSUE THREAD

**Issue by robjtede** [ixhbinphoenix/actix-session-surrealdb#2]
in both the readme and crate docs you have this snippet:

```
    HttpServer::new(move || {
        App::new()
            .wrap(
                SessionMiddleware::builder(session_store.clone(), Key::generate())
```

this is very subtly incorrect as it will generate a different key for each server worker

please amend it to

```
    let key = Key::generate();

    HttpServer::new(move || {
        App::new()
            .wrap(
                SessionMiddleware::builder(session_store.clone(), key.clone())
```

and publish a new release so users of this crate can copy-paste from the updated crates.io page

**Comment by ixhbinphoenix** [comment]
Thank you for opening an issue for this, would have never noticed that

**Comment by robjtede** [comment]
fixed the other occurence in #3
